src/pudl/analysis/record_linkage/eia_ferc1_splink_model_config.py

```diff
@@ -0,0 +1,93 @@
+"""The model parameters for the FERC1 to EIA splink record linkage model.
+
+This module enumerates the blocking rules as well as the comparison levels
+for the matching columns that are used in the FERC1 to EIA record linkage
+model.
+"""
+import splink.duckdb.comparison_level_library as cll
+import splink.duckdb.comparison_library as cl
+import splink.duckdb.comparison_template_library as ctl
+from splink.duckdb.blocking_rule_library import block_on
+
+blocking_rule_1 = "l.report_year = r.report_year and jaro_winkler_similarity(l.plant_name, r.plant_name) >= .7"
```

Comment:
TODO for me: update these blocking rules to use an alternative to `jaro_winkler_similarity` to improve model performance.